src/eventHandler.js

```diff
@@ -93,7 +93,20 @@ export default superclass => class extends superclass {
 			})
 		);
 
-		(eventRes || eventRes === null) && this._setTranslate([-pos, 0]);
+		if (eventRes || eventRes === null) {
+			this._setTranslate([-pos, 0]);
+		} else {
+			e.stop();
+			// release, animationEnd
+			touch.distance = 0;
+			touch.direction = null;
+			touch.destPos = 0;
+			this._adjustContainerCss("end");
+			touch.holding = false;
+			this._setPointerEvents();
+			touch.isTrusted = false;
+			this._setPhaseValue("end");
```

Comment:
@younkue How about grouping code into 2 groups. One group for `touch`, the other group is to call `method`?

Reply:
@younkue Are these methods affected by order?